pom.xml

```diff
@@ -55,10 +55,16 @@
     <basepom.failsafe.timeout>900</basepom.failsafe.timeout>
 
     <dep.plugin.compiler.version>3.8.0</dep.plugin.compiler.version>
+    <dep.plugin.dependency.version>3.1.2-hubspot-SNAPSHOT</dep.plugin.dependency.version>
```

Comment:
Maybe we should do our own release of the dependency plugin or something? This version stops us from releasing baspom:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-release-plugin:2.5.3:prepare (default-cli) on project baseepom: Can't release project due to non released dependencies :
[ERROR]     org.apache.maven.plugins:maven-dependency-plugin:maven-plugin:3.1.2-hubspot-SNAPSHOT:runtime
[ERROR] in project 'basepom' (com.hubspot:basepom:pom:25.7-SNAPSHOT)
```

Reply:
I think that this will also not be available to people compiling in an environment that doesn't have access to our nexus instance.